src/game/game.cpp

```diff
@@ -7614,6 +7603,9 @@ void Game::playerCreateMarketOffer(uint32_t playerId, uint8_t type, uint16_t spr
 		}
 
 		std::forward_list<Item *> itemList = getMarketItemList(it.wareId, stashmath, depotLocker);
+		if (itemList.empty()) {
+			return;
+		}
```

Comment:
with this here you can remove the if !itemList.empty below

Reply:
I didn't want to mess with this too much, as it's market code, but I'll do it.